static/app/components/environmentPageFilter.tsx

```diff
@@ -0,0 +1,92 @@
+import {useState} from 'react';
+import {withRouter, WithRouterProps} from 'react-router';
+import styled from '@emotion/styled';
+
+import {updateEnvironments} from 'sentry/actionCreators/globalSelection';
+import DropdownButton from 'sentry/components/dropdownButton';
+import MultipleEnvironmentSelector from 'sentry/components/organizations/multipleEnvironmentSelector';
+import {IconWindow} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import GlobalSelectionStore from 'sentry/stores/globalSelectionStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
+import space from 'sentry/styles/space';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
+
+type Props = {
+  router: WithRouterProps['router'];
+  /**
+   * Reset these URL params when we fire actions
```

Comment:
This is the tiniest nit pick. But you can probably keep these two lines on a single line, no need for the comment to be wrapped this hard.

doesn't really matter though 🤷